spec/components/trainees/confirmation/deferral_details/view.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
I don't think you meant to have this file here! Can just be deleted.

Reply:
Thanks